alternator/executor.cc

```diff
@@ -563,8 +585,10 @@ static future<rjson::value> fill_table_description(schema_ptr schema, table_stat
     auto tags_ptr = db::get_tags_of_table(schema);
 
     rjson::add(table_description, "TableName", rjson::from_string(schema->cf_name()));
-    // FIXME: take the tables creation time, not the current time!
-    size_t creation_date_seconds = std::chrono::duration_cast<std::chrono::seconds>(gc_clock::now().time_since_epoch()).count();
+
+    auto creation_time = get_table_creation_time(*schema);
+    auto creation_date = std::chrono::duration_cast<std::chrono::milliseconds>(creation_time.time_since_epoch()).count() / 1000.0;
```

Comment:
nitpick: the variable name "creation_date" is a little off - it's not really a date.

I wonder if instead of repeating this calculation twice, you shouldn't just have get_table_creation_time() return a double and that's it?

By the way, a few lines above you just used creation_time.time_since_epoch().count() without the duration_cast to milliseconds. The duration cast isn't needed because it's already in milliseconds, but it doesn't hurt either.